### ISSUE TITLE
[HLE Regression] liblv2.sprx and Libsysmodule.sprx together regression

### ISSUE THREAD

**Issue by Darkhost1999** [RPCS3/rpcs3#15518]
### Quick summary

In order to test this issue your config must look like
```
  Libraries Control:
    - liblv2.sprx:hle
    - libsysmodule.sprx:hle
```
The issue does not present itself with just 1 or the other of the sprx. Both are needed to reproduce the issue.

### Details

I observed some games I previously reported to work with no firmware installed inside RPCS3 at all became incapable of booting anymore as they both presented a fatal error: `F {PPU[0x1000001] Thread (ps3_init) [0x007ba880]} VM: Access violation writing location 0x0 (unmapped memory)`
At first, I started trying to single out which SPRX was at fault and found that the game worked when only 1 of the afflicted SPRX was HLE'd. So I had to find its accomplice. After the process of elimination, I found that lv2.sprx and sysmodule.sprx when working together but not separately cause these games to no longer boot when using the experimental HLE boot.

### Build with regression

RPCS3 v0.0.29-15508-1843a27c #14523

### Attach two log files

Battlefield 3
[Working.log](https://github.com/RPCS3/rpcs3/files/15140418/RPCS3.log)
[Broken.log](https://github.com/RPCS3/rpcs3/files/15140442/RPCS3.log)
Ratchet & Clank™: Quest For Booty
[Working.log](https://github.com/RPCS3/rpcs3/files/15140430/RPCS3.log)
[Broken.log](https://github.com/RPCS3/rpcs3/files/15140448/RPCS3.log)

### Attach capture files for visual issues

Not Applicable

### System configuration

Ryzen 7 7700x
GTX 1660 Super
Windows 11 Pro

### Other details

To keep in mind this issue isn't present under normal configurations as this issue is only presented with firmware overrides.
As HLE is the goal for the RPCS3 emulator it may be helpful to inform of regressions when utilizing HLE overrides.

**Comment by Darkhost1999** [comment]
![image](https://github.com/user-attachments/assets/31214d01-e58a-42cb-b8d8-0c04cbcb7a3d)
#16201
[RPCS3.log](https://github.com/user-attachments/files/17359406/RPCS3.log)
Good job libsysmodule still needs work but liblvl2 is working with BF3.
liblvl2 is not working with R&C QFB though.
[RPCS3.log](https://github.com/user-attachments/files/17363871/RPCS3.log)
Hope this gives more info on some LV2 fixes
I can redo the logs with only the afflicted sprx HLE'd if desired like the original post.

**Comment by digant73** [comment]
@Darkhost1999 is that still an issue?

**Comment by Darkhost1999** [comment]
Yes Battlefield 3 and  Ratchet & Clank™: Quest For Booty used to work without Sony's firmware installed at least to title screen and don't anymore

**Comment by Darkhost1999** [comment]
Well libsysmodule was fixed

**Comment by Darkhost1999** [comment]
Yes LLE of both worked.
The issue only occured when both libraries were HLE'd.
But since sysmodule was fixed the issue has been resolved. Liblvl2 on its own and libsysmodule on its own didn't present the issue. The regression is in a satisfied state of unable to replicate anymore.
